studio/components/interfaces/Database/EnumeratedTypes/EnumeratedTypes.tsx

```diff
@@ -0,0 +1,176 @@
+import { useState } from 'react'
+
+import SchemaSelector from 'components/ui/SchemaSelector'
+import {
+  EnumeratedType,
+  useEnumeratedTypesQuery,
+} from 'data/enumerated-types/enumerated-types-query'
+import { useProjectContext } from 'components/layouts/ProjectLayout/ProjectContext'
+import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
+import AlertError from 'components/ui/AlertError'
+import Table from 'components/to-be-cleaned/Table'
+import {
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  IconMoreVertical,
+  IconPlus,
+  IconSearch,
+  IconTrash,
+  Input,
+} from 'ui'
+import DeleteEnumeratedTypeModal from './DeleteEnumeratedTypeModal'
+import CreateEnumeratedTypeSidePanel from './CreateEnumeratedTypeSidePanel'
+import EditEnumeratedTypeSidePanel from './EditEnumeratedTypeSidePanel'
+
+const EnumeratedTypes = () => {
+  const { project } = useProjectContext()
+  const [search, setSearch] = useState('')
+  const [selectedSchema, setSelectedSchema] = useState('public')
+  const [showCreateTypePanel, setShowCreateTypePanel] = useState(false)
+  const [selectedTypeToEdit, setSelectedTypeToEdit] = useState<EnumeratedType>()
+  const [selectedTypeToDelete, setSelectedTypeToDelete] = useState<EnumeratedType>()
+
+  const { data, error, isLoading, isError, isSuccess } = useEnumeratedTypesQuery({
+    projectRef: project?.ref,
+    connectionString: project?.connectionString,
+  })
+  const enumeratedTypes =
+    search.length > 0
+      ? (data ?? []).filter(
+          (x) => x.schema === selectedSchema && x.name.toLowerCase().includes(search.toLowerCase())
+        )
+      : (data ?? []).filter((x) => x.schema === selectedSchema)
+
+  return (
+    <>
+      <div className="flex items-center justify-between mb-4">
+        <div className="flex items-center space-x-2">
+          <div className="w-[260px]">
```

Comment:
could just pass this className to SchemaSelector component.

Reply:
yup! makes sense